backend/src/zango/apps/shared/tenancy/management/commands/ws_makemigration.py

```diff
@@ -45,7 +45,15 @@ def add_arguments(self, parser):
     def handle(self, *app_labels, **options):
         is_test_mode = options["test"]
         tenant = options["workspace"]
-        tenant_obj = TenantModel.objects.get(name=tenant)
+        while True:
+            try:
+                tenant_obj = TenantModel.objects.get(name=workspace)
```

Comment:
it will throw an error, `UnboundLocalError: local variable 'workspace' referenced before assignment` for the first time